pom.xml

```diff
@@ -59,43 +58,47 @@
     <build>
         <plugins>
             <plugin>
-                <groupId>org.apache.felix</groupId>
-                <artifactId>maven-scr-plugin</artifactId>
-            </plugin>
-            <plugin>
-                <groupId>org.apache.sling</groupId>
-                <artifactId>maven-sling-plugin</artifactId>
-            </plugin>
-            <plugin>
-                <groupId>org.apache.felix</groupId>
-                <artifactId>maven-bundle-plugin</artifactId>
-                <extensions>true</extensions>
-                <configuration>
-                    <instructions>
-                        <Bundle-SymbolicName>org.apache.sling.distribution.core</Bundle-SymbolicName>
-                        <Export-Package>
-                            org.apache.sling.distribution.util,
-                            org.apache.sling.distribution.common,
-                            org.apache.sling.distribution.serialization,
-                            org.apache.sling.distribution.queue.spi,
-                            org.apache.sling.distribution.queue,
-                            org.apache.sling.distribution.log.spi,
-                            org.apache.sling.distribution.agent,
-                            org.apache.sling.distribution.agent.spi,
-                            org.apache.sling.distribution.packaging
-                        </Export-Package>
-                        <Import-Package>
-                            org.apache.http.nio.*;resolution:=optional,
-                            org.apache.http.impl.nio.client.*;resolution:=optional,
-                            org.apache.felix.hc.api;resolution:=optional,
-                            org.apache.sling.event.jobs;version="[1.5.0,3.0)",
-                            org.apache.sling.event.jobs.consumer;version="[1.1,2)",
-                            org.apache.sling.jcr.api;version="2.2.0",
-                            *
-                        </Import-Package>
-                    </instructions>
-                </configuration>
+                <groupId>biz.aQute.bnd</groupId>
+                <artifactId>bnd-maven-plugin</artifactId>
             </plugin>
+<!--             <plugin> -->
```

Comment:
You should remove the commented out code.

Reply:
fixed